### ISSUE TITLE
Add option for object score to be displayed at top of app

### ISSUE THREAD

**Issue by pawn002** [pawn002/color-pair-quick-iterator#90]
Similar to the perceptual contrast score and the wcag2 score, the object score should be up there as an option for users.
